indexer/packages/postgres/src/helpers/db-helpers.ts

```diff
@@ -8,6 +8,7 @@ import { rawQuery } from './stores-helpers';
 const layer2Tables = [
   'perpetual_positions',
   'fills',
+  'subaccount_usernames',
```

Comment:
@Christopher-Li IIUC this should be layer 2 right? since this depends on another table

Reply:
Yup